src/app.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  name: "InputFormats",
+  components: {
+    VueMoney: () => import("./components/money"),
+  },
+  props: {
+    type: {
+      type: String,
+      default: "money",
+    },
+  },
+  data() {
+    return {};
+  },
+
+  render(h) {
+    let template;
+
+    switch (this.type) {
+      case "money":
+        template = h("VueMoney", {});
+        break;
+      default:
+        template = "";
+    }
```

Comment:
Should be separated into another function that accepts your `this.type` and returns `h("VueMoney", {});` or `""`;

Your final code should be something like this: 
```javascript
const template = yourFunction(this.type);

return h("div", {}, [ template ]);
```

Reply:
Use meaningful names. What does mean `h` ?

Reply:
> Use meaningful names. What does mean `h` ?

Its createElement for render function